notebooks/MosvizExample.ipynb

```diff
@@ -129,7 +129,13 @@
     "    elif 's2d' in file_path:\n",
     "        spectra_2d.append(file_path)\n",
     "\n",
-    "images = [str(level3_path / 'mosviz_cutouts' / '227.fits')] * len(spectra_1d)"
+    "images = [str(level3_path / 'mosviz_cutouts' / '227.fits')] * len(spectra_1d)\n",
+    "\n",
+    "# *** OR ***\n",
+    "# If images are available in a cutouts directory, uncomment the following lines.\n",
+    "# Please make sure the number of cutouts matches the number of x1d and s2d files.\n",
```

Comment:
Hmm instead of a comment about "making sure," is it possible to actually add that check logic in a follow-up cell? Or is that too complicated?

Reply:
I guess it's possible, although I'm not sure if the way mosviz handles images will change with the refactor, making any change obsolete. That's my rational for doing this the easy way at least.

Reply:
That makes sense. Let's keep it as is.